x/session/keeper/keeper.go

```diff
@@ -60,12 +60,12 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 func (k Keeper) BeginBlocker(ctx sdk.Context) {
 	// ctx.BlockHeader().LastBlockId.Hash is the hash of the last block committed
 	hash := ctx.BlockHeader().LastBlockId.Hash
-	// ctx.BlockHeader().Height is the height of the current block being processed,
-	// decrementing it by 1 gives us the height of the last block committed.
-	height := ctx.BlockHeader().Height - 1
-	// If height is 0, ctx.BlockHeader().LastBlockId.Hash will be nil,
-	// fall back to an empty byte slice.
-	if height == 0 {
+	// ctx.BlockHeader().Height is the height of the last committed block.
+	height := ctx.BlockHeader().Height
+	// When height is 1, ctx.BlockHeader().LastBlockId.Hash will not be populated,
+	// and leads to a panic when trying to save the hash in the store.
```

Comment:
@red-0ne This is a good comment but if we copy-paste (and reword) what marko told you as a comment, there will be no ambiguity fo anyone.